src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -115790,13 +115790,13 @@ void M0<T>(T x0)
 ";
 
             CreateCompilation(source, options: WithNullableEnable()).VerifyDiagnostics(
-                // (6,19): error CS0150: A constant value is expected
+                // (6,19): error CS9135: A constant value of type 'T' is expected
```

Comment:
>// (6,19): error CS9135: A constant value of type 'T' is expected [](http://example.com/codeflow?start=16&length=65)

For a generic type parameter should probably stick to the old message too.

Reply:
Done!